### PR TITLE
relative to classpath vs. src

### PR DESCRIPTION
**Pull request by ghchinoy**
This is an alternate way of addressing the oauth.properties file in AuthUtil.java - relative to the classloader instead of directly to a file path. This method helps when deploying to AppEngine.

Still works with mvn jetty:run and mvn appengine:devserver and when pushed to AppEngine.

(probably don't need the edits to .gitignore)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,14 @@
...
+
+# Local Eclipse #
+.settings
+.configuration
+.classpath
+
+# Maven output
+target
+
+# Personal config
```
Comment (mimming) [inline .gitignore]: I don't think we want to add this to the main project. Please remove the references to myoauth.properties

File: src/main/java/com/google/glassware/AuthUtil.java [hunk 2]
```diff
@@ -41,13 +46,24 @@
...
       + "https://www.googleapis.com/auth/userinfo.profile";
+  private static final Logger LOG = Logger.getLogger(AuthUtil.class.getSimpleName());
 
   /**
    * Creates and returns a new {@link AuthorizationCodeFlow} for this app.
    */
   public static AuthorizationCodeFlow newAuthorizationCodeFlow() throws IOException {
-    FileInputStream authPropertiesStream =
-        new FileInputStream("./src/main/resources/oauth.properties");
+    //FileInputStream authPropertiesStream =
```
Comment (mimming) [inline src/main/java/com/google/glassware/AuthUtil.java]: Prune the removed / commented out code

File: src/main/java/com/google/glassware/AuthUtil.java [hunk 3]
```diff
@@ -41,13 +46,24 @@
...
+    //    new FileInputStream("./src/main/resources/oauth.properties");
+    URL resource = AuthUtil.class.getResource("/myoauth.properties");
+    File propertiesFile = new File("./src/main/resources/myoauth.properties");
+    try {
+      propertiesFile = new File(resource.toURI());
+      //LOG.info("Able to find oauth properties from file.");
```
Comment (mimming) [inline src/main/java/com/google/glassware/AuthUtil.java:59]: One more commented line to remove :)

File: src/main/java/com/google/glassware/AuthUtil.java [hunk 4]
```diff
@@ -41,13 +46,24 @@
...
 
   /**
    * Creates and returns a new {@link AuthorizationCodeFlow} for this app.
    */
   public static AuthorizationCodeFlow newAuthorizationCodeFlow() throws IOException {
-    FileInputStream authPropertiesStream =
-        new FileInputStream("./src/main/resources/oauth.properties");
+    //FileInputStream authPropertiesStream =
+    //    new FileInputStream("./src/main/resources/oauth.properties");
+    URL resource = AuthUtil.class.getResource("/myoauth.properties");
```
Comment (mimming) [inline src/main/java/com/google/glassware/AuthUtil.java]: change this and the next line to oauth.properties

**Comment by mimming** [comment]
Thanks for the pull request (and for filling out the CLA). This looks like a great change, but I'll just need a little bit of cleanup before I can merge it in.

**Comment by ghchinoy** [comment]
Can do, thank you for looking at this!

**Pull request merged**